### ISSUE TITLE
Grav 1.7 RC.6 Admin panel can not open /admin/config

### ISSUE THREAD

**Issue by FBachofner** [getgrav/grav-plugin-admin#1840]
When I click on "Configuration" on the left side of the Admin panel (which would lead to example.com/admin/config ) I get a "Secure Connection Failed" error . . . "The page you are trying to view cannot be shown because the authenticity of the received data could not be verified."

The site has a valid/functioning Let'sEncrypt SSL certificate and all pages should be https according to my nginx config for this doman's virtual server.

"Manually" attempting to open the link (i.e. paste the /admin/config URL into the address bar) also fails.

Meanwhile, **no** other links in the Admin panel seem to have problems. (i.e. /admin/pages/, /admin/accounts/users, /admin/plugins, /admin/themes, /admin/tools  ALL work)

If I make an arbitrary folder (i.e. example.com/test ) and place an index.html  "hello world -type" test file, the nginx server is able to https serve such page with no issue.

The public side of the Grav site is **also** able to serve its secure pages (a couple test pages, so far) without incident as well.

Additional details:
- The nginx .conf file does not seem to have anything in it which would preclude the opening of /admin/config
- The SSL certificate is not configured to exclude any folders (I don't think that is possible anyway)
- There is no firewall rule which would cause such a problem

Is this a Grav issue or do I have something configured wrong in nginx or elsewhere?

Thanks in advance!

**Comment by rhukster** [comment]
Can you screenshot this error? Sounds like a webserver/browser issue.  Don't think that's a grav error.

**Comment by FBachofner** [comment]
Hi Andy / @rhukster:

I thought the same thing, so I tried a different computer with a **_very_** current version of Firefox (73.0.1).  On that computer, Admin/Configuration worked, but then a bunch of pages descendant of /admin/config did NOT and/or experienced intermittent problems.

I noticed that my user profile (the ONLY user, effectively the administrator -- although no groups exist) had all permissions "not set" from the "configuration" section on down.  I have set them all to "allowed."

Unfortunately, I still can not get into Admin/Configuration from the first computer and have sporadic connections from the second.  I get that intermittent issues are the hardest with IT, but this server seems to have no problems with my other CMS (ModX) nor with hand-coded sites.  I think the issue is somehow related to Grav. [Can Grav caching affect this somehow? ]  Is it possible I am missing a PHP dependency?

Requested screenshot is attached.
![Grav 1 7 0_rc6 Secure Connection Failed](https://user-images.githubusercontent.com/22858382/75396630-d6855100-58a9-11ea-88ad-4a7cf2893ee2.png)
